CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 3.0.0-alpha.15 (2019-06-14)
+
+
+### Bug Fixes
+
+* Prism's build process received some tweaks, but there's more work to do #352
+
+### Features
+
+* Prism now has got a static example fallback in case the `dynamic` flag is not enabled #347
+* Prism can now validate servers #351
+
+### Chores
```

Comment:
Thank yiou but what I meant was that we dont need to list chores. Changelogs are for changes to the software, to let the user know what new features they are getting, what bugs were fixed, and what breaking changes happened. 

The users don't care if we cleaned our teeth and mowed the lawn during this time, just the prism changes.

Reply:
ditched

Reply:
Ah ok fantastic, removing then